src/renderer/components/Modal/ModalExportWallets.vue

```diff
@@ -0,0 +1,265 @@
+<template>
+  <ModalWindow
+    :allow-close="true"
+    :message="$t('MODAL_EXPORT_WALLETS.INSTRUCTIONS')"
+    container-classes="w-2/5"
+    @close="emitClose"
+  >
+    <section class="flex flex-col">
+      <h2 class="mb-4">
+        {{ $t('MODAL_EXPORT_WALLETS.TITLE') }}
+      </h2>
+
+      <ListDivided>
+        <ListDividedItem
+          v-for="[option, values] in Object.entries(options)"
+          :key="option"
+          :item-label-class="!!values.isDisabled ? 'opacity-50' : ''"
+          :item-value-class="!!values.isDisabled ? 'opacity-50 cursor-not-allowed' : ''"
+          :label="`${$t('MODAL_EXPORT_WALLETS.OPTIONS.' + strings_snakeCase(option).toUpperCase())}`"
+        >
+          <ButtonSwitch
+            ref="option"
+            :is-disabled="!!values.isDisabled"
+            :is-active="values.active"
+            class="ml-3"
+            @change="toggleOption(option)"
+          />
+        </ListDividedItem>
+      </ListDivided>
+
+      <h4 class="mt-5">
+        {{ $t('MODAL_EXPORT_WALLETS.ADVANCED') }}
+      </h4>
+
+      <ListDivided>
+        <ListDividedItem
+          v-for="option in Object.keys(advancedOptions)"
+          :key="option"
+          :label="`${$t('MODAL_EXPORT_WALLETS.OPTIONS.' + strings_snakeCase(option).toUpperCase())}`"
+        >
+          <ButtonSwitch
+            ref="option"
+            :is-active="advancedOptions[option]"
+            class="ml-3"
+            @change="toggleOption(option, true)"
+          />
+        </ListDividedItem>
+      </ListDivided>
+
+      <div class="flex mt-5">
+        <ButtonGeneric
+          :label="$t('MODAL_EXPORT_WALLETS.CANCEL')"
+          class="mr-5"
+          @click="emitClose"
+        />
+        <ButtonGeneric
+          :disabled="isExporting || $v.options.$invalid"
+          :label="$tc('MODAL_EXPORT_WALLETS.EXPORT', wallets.length, { count: wallets.length })"
+          @click="exportWallets"
+        />
+      </div>
+    </section>
+  </ModalWindow>
+</template>
+
+<script>
+import { isNull, omitBy, uniqBy } from 'lodash'
+import ModalWindow from '@/components/Modal/ModalWindow'
+import { ButtonGeneric, ButtonSwitch } from '@/components/Button'
+import { ListDivided, ListDividedItem } from '@/components/ListDivided'
+
+export default {
+  name: 'ModalExportWallets',
+
+  components: {
+    ButtonGeneric,
+    ButtonSwitch,
+    ListDivided,
+    ListDividedItem,
+    ModalWindow
+  },
+
+  data () {
+    return {
+      isExporting: false,
+      options: {
+        excludeUnnamed: {
+          active: false,
+          filter: el => el.name && el.name.length
+        },
+        excludeEmpty: {
+          active: false,
+          filter: el => el.balance
+        },
+        excludeLedger: {
+          active: false,
+          isDisabled: !this.isLedgerConnected,
```

Comment:
The `data` can not access the `computed` properties, you need to use the `watch`:

```js
watch: {
  isLedgerConnected: {
    handler(val) {
      this.options.excludeLedger.isDisabled = !val
    },
    immediate: true
  }
}
```